### DISCUSSION TITLE
ltsp image / 235: eval: Syntax error: Unterminated quoted string

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#706] [General]
Solved!
Excuse-me!

Hello!
I update and upgrade Xubuntu 20.04 LTS. Add on more user with automatic logon using command: base64, and insert this on flie /etc/ltsp.conf with the Mac address. Where we have more 12 users that boot system normally. So, After add the user and on server and try generate new image, it gives me the error:

`root@beneltsp:~# !287
ltsp image /
/usr/sbin/ltsp: 235: eval: Syntax error: Unterminated quoted string
`
Please, How can I fix this?

Thank you

Douglas

**Comment by alkisg** [comment]
Hi Douglas, just fix your /etc/ltsp/ltsp.conf, you forgot to close a quote there (").

**Reply by doguibnu** [reply]
Hello @alkisg 
Yes! You are right.
Thanks so much for attention.
